### PR TITLE
SAK-40530: Roster > UI improvements and new features

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/SAK-40530

See list of changes and additions, screenshots, and test plan in the JIRA.

### CODE REVIEW

**Comment by bjones86** [comment]
The Codacy complaints here are all invalid.

File: roster2/src/handlebars/members_header.handlebars [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <div class="roster-table-header-group">
         <div class="roster-table-row">
             {{#if viewPicture}}
-            <div class="roster-table-cell">{{translate 'facet_picture'}}</div>
+            <div class="roster-table-cell" id="roster-picture-header-cell">{{translate 'facet_picture'}}</div>
```
Comment (ottenhoff) [inline roster2/src/handlebars/members_header.handlebars]: does this id get repeated for each member? if yes, id should be unique

Comment (bjones86) [reply roster2/src/handlebars/members_header.handlebars]: Not repeated; this file defines the "header" row of the "table", which is only visible in spreadsheet/table/list view mode.

Comment (ottenhoff) [reply roster2/src/handlebars/members_header.handlebars]: 👍

**Pull request merged**